tools/ci/azure/cleanup_win10.yml

```diff
@@ -72,26 +72,26 @@ steps:
   ignoreLASTEXITCODE: true
 
 - powershell: |
-    $channelFolders = @('Edge', 'Edge SxS', 'Edge Dev', 'Edge Beta')
-    foreach ($folder in $channelFolders) {
-      Write-Host "Deleting regkeys for $folder folder"
-      Remove-Item -Path "HKLM:\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Microsoft $folder" -Recurse -Force -ErrorAction SilentlyContinue
-      Remove-Item -Path "HKCU:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Microsoft $folder" -Recurse -Force -ErrorAction SilentlyContinue
+    $channels = @('Edge', 'Edge Beta', 'Edge Dev', 'Edge Canary')
```

Comment:
Canary was SxS before, was that an intentional change?

Reply:
Not intentional, good catch, thank you!